### PR TITLE
Construct server

### PR DESCRIPTION
**Pull request by papakapa**
swap all console.log to console.error
add const for mongo.connect

### CODE REVIEW

File: serv.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const config = require('config');
+const mongoose = require('mongoose');
+const Track = require('./models/Track');
+const Author = require('./models/Author');
+
+const PORT = config.get('port') || 63341;
+
+const app = express();
+
+mongoose.connect("mongodb+srv://kiryl:1111@cluster0.2ltpu.mongodb.net/music?retryWrites=true&w=majority", {
```
Comment (m4eryk) [inline serv.js:11]: move url to const

File: serv.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    useUnifiedTopology: true
+});
+
+app.use(express.json({extended: true}))
+
+app.get('/', (req, res) => {
+    res.send('Hi');
+})
+app.post('/', (req, res) => {
+    console.log(req.body);
```
Comment (m4eryk) [inline serv.js:24]: remove all console.log

File: serv.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            if (err || !track) {
+                const track = new Track({
+                    link: song.links[i],
+                    poster: song.posters[i],
+                    name: song.names[i],
+                    author: song.author[i]
+                });
+                track.save().then(() => console.log('Track Created'));
+            } else {
+                console.log('Track exist or other error', err);
```
Comment (m4eryk) [inline serv.js:37]: use console.error

**Review by m4eryk — CHANGES_REQUESTED** [review]

File: config/default.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "PORT": 63341,
```
Comment (papakapa) [inline config/default.json:2]: add SCREAMING SNAKE CASE

File: config/default.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+  "MONGO_URL" :"mongodb+srv://kiryl:1111@cluster0.2ltpu.mongodb.net/music?retryWrites=true&w=majority"
+}
+
```
Comment (papakapa) [inline config/default.json:5]: add new line in the of file

**Pull request merged**